docs/pages/database-access/getting-started.mdx

```diff
@@ -0,0 +1,157 @@
+description: Getting started with Teleport Database Access and AWS Aurora PostgreSQL
+---
+
+# Getting Started
+
+In this getting started guide we will use Teleport Database Access to connect
+to a PostgreSQL flavored AWS Aurora database.
+
+Here's an overview of what we will do:
+
+1. Configure AWS Aurora database with IAM authentication.
+2. Download and install Teleport and connect it to the Aurora database.
+3. Connect to the Aurora database via Teleport.
+
+## Step 1/3. Setup Aurora
+
+In order to allow Teleport connections to an Aurora instance, it needs to support
+IAM authentication.
+
+If you don't have a database provisioned yet, create an instance of an Aurora
+PostgreSQL in the [RDS control panel](https://console.aws.amazon.com/rds/home).
+Make sure to choose "Standard create" database creation method and enable
+"Password and IAM database authentication" in the Database Authentication dialog.
+
+For existing Aurora instances, the status of IAM authentication is displayed on
+the Configuration tab and can be enabled by modifying the database instance.
+
+Next, create the following IAM policy and attach it to the AWS user or service
+account. Teleport Database Service will need to use credentials of this AWS user
+or service account in order to use this policy.
+
+```json
+{
+   "Version": "2012-10-17",
+   "Statement": [
+      {
+         "Effect": "Allow",
+         "Action": [
+             "rds-db:connect"
+         ],
+         "Resource": [
+             "arn:aws:rds-db:<region>:<account-id>:dbuser:<resource-id>/*"
+         ]
+      }
+   ]
+}
+```
+
+This policy allows any database account to connect to the Aurora instance specified
+with resource ID using IAM auth.
+
+<Admonition type="note" title="Resource ID">
+Database resource ID is shown on the Configuration tab of a particular
+database instance in RDS control panel, under "Resource id". For regular
+RDS database it starts with `db-` prefix. For Aurora, use the database
+cluster resource ID (`cluster-`), not individual instance ID.
+</Admonition>
+
+Finally, connect to the database and create a database account with IAM auth
+support (or update an existing one). Once connected, execute the following
+SQL statements to create a new database account and allow IAM auth for it:
+
+```sql
+CREATE USER alice;
+GRANT rds_iam TO alice;
+```
+
+For more information about connecting to the PostgreSQL instance directly,
+see Amazon [documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ConnectToPostgreSQLInstance.html).
+
+## Step 2/3. Setup Teleport
+
+Teleport Database Access is available starting from `6.0.0` release.
+
+Download the appropriate version of Teleport for your platform from the table
+below, or visit our [downloads page](https://goteleport.com/teleport/download).
+
+{!docs/pages/preview/releases-table.mdx!}
+
+<Admonition type="warning" title="Version Warning">
+Note, pre-releases are not suitable for production usage!
```

Comment:
It says warning, but the text says 'note'. Also, if you switch to active here, see the difference:

`Make sure not to use pre-releases in production.`